README.md

```diff
@@ -1 +1,37 @@
-# Hello-World
\ No newline at end of file
+# Project Name
+Hello World
+
+> Description the project.
+Exersice for creating repos, branches & commits.
+
+## Built With
+
```

Comment:
- [ ] [optional] It would be nice to add all tools used HTML and CSS. Also, consider adding how to clone and run your project.